aws-iot-mitigationaction/aws-iot-mitigationaction.json

```diff
@@ -0,0 +1,248 @@
+            "ERROR",
+            "WARN"
+          ],
+          "description": " Specifies which types of information are logged."
+        },
+        "RoleArnForLogging": {
+          "description": " The ARN of the IAM role used for logging.",
+          "type": "string",
+          "minLength": 20,
+          "maxLength": 2048
+        }
+      },
+      "required": [
+        "LogLevel",
+        "RoleArnForLogging"
+      ]
+    },
+    "PublishFindingToSnsParams": {
+      "type": "object",
+      "description": "Parameters, to define a mitigation action that publishes findings to Amazon SNS. You can implement your own custom actions in response to the Amazon SNS messages.",
+      "properties": {
+        "TopicArn": {
+          "type": "string",
+          "description": "The ARN of the topic to which you want to publish the findings.",
+          "minLength": 20,
+          "maxLength": 2048
+        }
+      },
+      "required": [
+        "TopicArn"
+      ]
+    },
+    "ReplaceDefaultPolicyVersionParams": {
+      "type": "object",
+      "description": "Parameters to define a mitigation action that adds a blank policy to restrict permissions.",
+      "properties": {
+        "TemplateName": {
+          "type": "string",
+          "enum": [
+            "BLANK_POLICY"
+          ]
+        }
+      },
+      "required": [
+        "TemplateName"
+      ]
+    },
+    "UpdateCACertificateParams": {
+      "type": "object",
+      "description": "Parameters to define a mitigation action that changes the state of the CA certificate to inactive.s",
+      "properties": {
+        "Action": {
+          "type": "string",
+          "enum": [
+            "DEACTIVATE"
+          ]
+        }
+      },
+      "required": [
+        "Action"
+      ]
+    },
+    "UpdateDeviceCertificateParams": {
+      "type": "object",
+      "description": "Parameters to define a mitigation action that changes the state of the device certificate to inactive.",
+      "properties": {
+        "Action": {
+          "type": "string",
+          "enum": [
+            "DEACTIVATE"
+          ]
+        }
+      },
+      "required": [
+        "Action"
+      ]
+    }
+  },
+  "properties": {
+    "ActionName": {
+      "description": "A unique identifier for the mitigation action.",
+      "type": "string",
+      "pattern": "[a-zA-Z0-9:_-]+",
+      "minLength": 1,
+      "maxLength": 128
+    },
+    "RoleArn": {
+      "type": "string"
+    },
+    "Tags": {
+      "type": "array",
+      "maxItems": 50,
+      "uniqueItems": true,
+      "insertionOrder": false,
+      "description": "An array of key-value pairs to apply to this resource.",
+      "items": {
+        "$ref": "#/definitions/Tag"
+      }
+    },
+    "ActionParams": {
+      "$ref": "#/definitions/ActionParams"
+    },
+    "MitigationActionArn": {
+      "type": "string"
+    },
+    "MitigationActionId": {
+      "type": "string"
+    },
+    "CreationDate": {
```

Comment:
As discussed, let's remove the CreationDate

Reply:
Done